corehq/apps/hqadmin/templates/hqadmin/raw_couch.html

```diff
@@ -46,7 +46,8 @@
 
         {% if doc_id %}
         <div class="container-fluid">
-            <div class="col-xs-2">
```

Comment:
The columns overlapped in smaller windows, and not all the space on the right was being used.